fastlane/lib/fastlane/auto_complete.rb

```diff
@@ -0,0 +1,16 @@
+module Fastlane
+  # Enable tab auto completion
+  class AutoComplete
+    def self.execute
+      path = File.join(Fastlane::Helper.gem_path('fastlane'), 'lib', 'assets', 'completions')
+
+      fastlane_conf_dir = "~/.fastlane"
+      `mkdir -p #{fastlane_conf_dir} && cp -r #{path} #{fastlane_conf_dir}`
```

Comment:
How would you feel about starting off by saying what the command is about to do for the user, and prompting them for agreement, before copying files into their home dir?

```
This will copy a shell script into ~/.fastlane/completions that provides the command tab completion.
Sound good?
(y / n) >  y

Great! That's all set.
To use auto complete ...
```